e2e/tests/regression/test_replica.robot

```diff
@@ -14,6 +14,15 @@ Resource    ../keywords/workload.resource
 Test Setup    Set test environment
 Test Teardown    Cleanup test resources
 
+*** Keywords ***
+Create deployment ${deployment_id} with volume ${volume_id}
+    ${volume_id}=  Convert To String  ${volume_id}
+    Create persistentvolume for volume ${volume_id}
+    Create persistentvolumeclaim for volume ${volume_id}
+    ${deployment_name}=    generate_name_with_suffix    deployment    ${deployment_id}
+    ${pvc_name}=    generate_name_with_suffix    volume    ${volume_id}
+    create_deployment    ${deployment_name}   ${pvc_name}
+
```

Comment:
The keyword `Create deployment ${deployment_id} with volume ${volume_id}` appears in two files:
1. [test_replica.robot](https://github.com/longhorn/longhorn-tests/blob/b431dafb10250af0f595061fc63b35e270fcfe1c/e2e/tests/regression/test_replica.robot#L18-L24)
2. [test_backup_listing.robot](https://github.com/longhorn/longhorn-tests/blob/b431dafb10250af0f595061fc63b35e270fcfe1c/e2e/tests/negative/test_backup_listing.robot#L50-L56)

Since these keywords are identical, they should be moved to `../keywords/deployment.resource` to avoid code duplication.

Reply:
```robot
*** Keywords ***
Create deployment ${deployment_id} with volume ${volume_id}
    ${volume_id}=  Convert To String  ${volume_id}
    Create persistentvolume for volume ${volume_id}
    Create persistentvolumeclaim for volume ${volume_id}
    ${deployment_name}=    generate_name_with_suffix    deployment    ${deployment_id}
    ${pvc_name}=    generate_name_with_suffix    volume    ${volume_id}
    create_deployment    ${deployment_name}   ${pvc_name}
```
I did not move this keyword to the resource file because the `Create deployment ${deployment_id} ...` contains the keywords from `volume.resource` and `deployment.resource`.
Does it allow for including another resource in a resource?

Reply:
I have tested it with your PR, and it looks good without any errors.

test report :  
[log.tar.gz](https://github.com/user-attachments/files/19860801/log.tar.gz)

![Screenshot_20250423_123223](https://github.com/user-attachments/assets/c8ce9e36-262e-4826-a3d0-52b98fb4f86d)